src/main/resources/templates/fragments/rightsAndResponsibilitiesFragments.html

```diff
@@ -458,3 +458,10 @@ <h3 class="text--small">SNAP/EBT Card: Multiple Card Replacements</h3>
         satisfactory immigration status.</p>
   </div>
 </th:block>
+<th:block th:fragment="privacyPolicy">
+  <div class="text--small">
+    <p>I understand that the information I’ve submitted is housed outside of the State of Maryland,
```

Comment:
Why not message properties keys here?

Reply:
That's the ideal. Vraj choose to hard code it because there is so much text and I just kept going with that. It is one area that we will have to revisit if we do include other languages.